src/components/firebase.js

```diff
@@ -0,0 +1,18 @@
+import firebase from 'firebase/app'
```

Comment:
I am not shure that you should push this file to github? You are not using firebase atm, but doesn't this leak sensitive information?

Reply:
You are right, I will folllow up on this one!